semeio/workflows/localisation/local_script_lib.py

```diff
@@ -344,7 +348,7 @@ def add_ministeps(
                     )
             elif impl_type == ErtImplType.SURFACE:
                 if corr_spec.surface_scale is not None:
-                    surface = Surface(corr_spec.surface_scale.filename)
+                    surface = Surface(corr_spec.surface_scale.surf_filename)
```

Comment:
Now I have checked it. What seems to work is: the function for the node: get_init_file_fmt() The function get_enkf_infile() returns only None both when running the localisation prior to creating initial ensemble and after the initial ensemble is created. I will update the code to use  get_init_file_fmt() and remove the filename keywword under surface_scale keyword.

Reply:
I have now rewritten the code to avoid specifying surface file in localisation config. But there are cases where I still need to supply a relative file path to the location of the surface files in the sub keyword INIT_FILES which is part of the SURFACE keyword in ERT config file. There are no way to get the file in the sub keyword BASE_SURFACE which have file path relative to CONFIG_PATH The files in the INIT_FILES subkeywords are often located in other directories. This is why we still must be able to specify a (relative) file path as a keyword in surface_scale

Reply:
Explanation of the issue:
libres seems not to have any function to retrieve the filename for the surface specified in the sub-keyword BASE_SURFACE in keyword SURFACE in ERT config file. There are then two alternatives to be able to get the size of the data for the SURFACE node:
1. Let the user specify the same file also in the localisation config file as in the ERT config file
2. Try a workaround using the libres function that get the surface file name from the INIT_FILES sub keyword of keyword SURFACE. The problem with this is:
   - The file name may be different from the one used in the BASE_SURFACE sub keyword, at least the file path is 
      usually different.
   -  The filename in INIT_FILES may have a %d as part of the name while the filename in BASE_SURFACE does not.
   
   If the only difference between the file name in INIT_FILES and BASE_SURFACE is the %d and the file path, this ca be handled by supplying the file path and replace the %d with 0.
   
What is preferred way of handling this?
1. Let the user supply a relative file path in localisation config file and create the filename we need from INIT_FILES since we cannot get the filename in BASE_SURFACE?
 2. Supply the filename specified in ERT config for BASE_SURFACE once more in the localisation config file?

Reply:
I have reverted this back to the original solution where the surface file is specified in localisation config file (should be the same as the file specified in the ERT keyword SURFACE under the sub keyword BASE_SURFACE.